hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -99,7 +99,7 @@ protected String name() {
   @Override
   public boolean dropTable(TableIdentifier identifier, boolean purge) {
```

Comment:
Do we need to add `validateTableIsIceberg` in dropTable then catch it and return false?

Reply:
No, it is thrown by `TableOperations` when the table is refreshed to check whether it exists at all.